src/EnergyPlus/ScheduleManager.cc

```diff
@@ -2776,7 +2765,13 @@ namespace ScheduleManager {
         //  so, current date, but maybe TimeStep added
 
         // Hourly Value
-        int thisHour = ThisHour + state.dataEnvrn->DSTIndicator;
+        int thisHour;
```

Comment:
Initialized "thisHour" outside the scope of the if statement. Its value is changed based on whether the UseDaylightSaving boolean has a "true" or a "false" value.